dronekit/__init__.py

```diff
@@ -1316,7 +1316,7 @@ def listener(self, name, msg):
                         self._params_last = monotonic.monotonic()
                         self._params_duration = start_duration
                     self._params_set[msg.param_index] = msg
-                self._params_map[msg.param_id] = msg.param_value
+                self._params_map[msg.param_id.rstrip(b'\x00').decode('utf8')] = msg.param_value
```

Comment:
I wonder if we should be stripping these at the pymavlink level.

Reply:
Probably yes. I will look into it.